packages/trader/src/Services/Helpers/__tests__/process-data.js

```diff
@@ -1,31 +0,0 @@
-import { expect } from 'chai';
```

Comment:
Why this file is removed?

Reply:
helpers re-added to shared package services

Reply:
as we talked because they are orphan files i removed them again.